config/types/disks.go

```diff
@@ -43,12 +43,12 @@ type Disk struct {
 }
 
 type Partition struct {
-	Label    string `yaml:"label"`
-	Number   int    `yaml:"number"`
-	Size     string `yaml:"size"`
-	Start    string `yaml:"start"`
-	GUID     string `yaml:"guid"`
-	TypeGUID string `yaml:"type_guid"`
+	Label    *string `yaml:"label"`
+	Number   int     `yaml:"number"`
+	Size     string  `yaml:"size"`
+	Start    string  `yaml:"start"`
```

Comment:
This and the one above now have a different semantics, what do we do?

Reply:
Ooops forgot the docs change. This is a backwards compatible change* so it shouldn't be a problem

* except for configs with partitions < 1M or not aligned to MBs.

Reply:
But the one above (size) now gives a different a different result (sectors vs MiB), isn't it?

Reply:
But they should be equivalent. I think it's fine if a newer version of `ct` emits an equivalent but different config. Otherwise we'd have to use deprecated fields.

Reply:
You are right, I'm sorry my brain missed the conversion logic in both old and new code.